benchmark/very_long_statment.py

```diff
@@ -0,0 +1 @@
x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z
```

Comment:
Same here, I would generate these, not check them in.

Reply:
i made option to add generating codes.